src/cache/redisCache.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable, Inject } from '@nestjs/common';
+import { Cache } from 'cache-manager';
+import { CACHE_MANAGER } from '@nestjs/cache-manager';
+
+@Injectable()
+export class RedisCacheService {
+  constructor(@Inject(CACHE_MANAGER) private readonly cache: Cache) {}
+
+  async get(key: string): Promise<any> {
+    return await this.cache.get(key);
+  }
+
+  async set(key: string, value: any) {
+    await this.cache.set(key, value);
+  }
```

Comment:
Consider using generics to specify the types

```suggestion
  async get<T>(key: string): Promise<T> {
    return await this.cache.get(key);
  }

  async set<T>(key: string, value: T) {
    await this.cache.set(key, value);
  }
```

this way you can use type inference when calling your methods.

Example: [Playground](https://www.typescriptlang.org/play?#code/MYGwhgzhAEDCbABYFMDKyBOA3AlsZ0A3gFDTTAD2AdhAC4YCuwtFGAFAA4Y5Zi0EZkYACbUQAT3IIUALmhgq4gJREAvsVLyI4qsGgBzZLQA8AFQB8bANbJxcutyr6lcgAoYKAWxwRkZ80SaZIK0DBhU8gDuYDi00LSIPgB0wNLISYa01rZKANya6pqQOnq+JhbZdtAOOE4ANNC8IAzIcqYqJGRkYNGx8YkQKWlJZZUNTS15BcSFlDRxqUhomLj40AC80FTIkXBp6Nh4yGyEqlPExbrQAGYMurQ41PHIdGwdGmQgRtVeyACqvgwAElhPZ6LV9PlNHM6I0wM0CJsejEFvsVkcMkY2AByBiAkHYpSaAD0xK60AAegB+D5whFJFgAGQokUw8F8byhZBhcTxmBBGyiKKkSwOq3SmWMNScllx+OEhPyZFJ5OptL5wOEDIozNZGHZx3O3OosI1IIATGDHPpBci+osUGKMZkcWaFVNlWSumrNBBfgD+cJbb1UaL0fhMVlsRQEpgA5rFSSyWrVEA)

You shouldn't use `any` as a type, prefer using `unknown`